plugins/cloudbuild/src/api/CloudbuildClient.ts

```diff
@@ -114,4 +91,23 @@ export class CloudbuildClient implements CloudbuildApi {
       'https://www.googleapis.com/auth/cloud-platform',
     );
   }
+
+  private async request(
+    url: string,
+    method: string = 'GET',
+  ): Promise<Response> {
+    const requestHeaders = {
+      Accept: '*/*',
+      Authorization: `Bearer ${await this.getToken()}`,
+      ...(process.env.NODE_ENV === 'production'
+        ? {
+            'User-Agent': `backstage/cloudbuild/${packageinfo.version}`,
```

Comment:
It's not possible to override the UA header in chrome. Is there a different header that could be used to signal this?

Reply:
i will test the usage of a custom header for this. I'll update in some days to confirm I can change it instead of using the User-Agent.